testdata/testinput5

```diff
@@ -2337,6 +2337,157 @@
 
 # End caseless restrict tests
 
+# TESTS for PCRE2_EXTRA_TURKISH_CASING - again, tests with and without.
+
+/i/i,utf
+    i
+    I
+    \x{0130}
+    \x{0131}
+
+/i/i,utf,turkish_casing
+    i
+    I
+    \x{0130}
+    \x{0131}
+
+/I/i,utf
+    i
+    I
+    \x{0130}
+    \x{0131}
+
+/I/i,utf,turkish_casing
+    i
+    I
+    \x{0130}
+    \x{0131}
+
+/\x{0130}/i,utf
+    i
+    I
+    \x{0130}
+    \x{0131}
+
+/\x{0130}/i,utf,turkish_casing
+    i
+    I
+    \x{0130}
+    \x{0131}
+
+/\x{0131}/i,utf
+    i
+    I
+    \x{0130}
+    \x{0131}
+
+/\x{0131}/i,utf,turkish_casing
+    i
+    I
+    \x{0130}
+    \x{0131}
+
+/[i]/i,utf
```

Comment:
I think this will be optimized out into an OP_CHARI unless there are more characters in the class

Reply:
The intention is just to observe & test the behaviour of all those edge-cases and optimizations, and check they work correctly. I don't particularly mind what the compiled OPcodes are, as long it matches the correct characters.